4001-5000/LIT4117Interpretation.xml

```diff
@@ -61,6 +61,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <revisionDesc>
          <change who="PL" when="2016-11-17">Created XML record from Ethio authority google spreadsheet</change>
          <change who="DR" when="2017-04-01">Added keywords and adjustments</change>
+         <change who="DN" when="2023-06-01">Added internal structure and abstract</change>
```

Comment:
Just a question for me to learn: "Added internal structure", that means, you added subdivisions and chapters to the work record?